README.md

```diff
@@ -10,14 +10,20 @@ _* Function eXecution_
 
 ## Install
 
-```bash
-go install github.com/antonmedv/fx@latest
+```console
+$ go install github.com/antonmedv/fx@latest
```

Comment:
Github has this nice feature where you can click a button to copy block of code. Let’s not add $ signs.

Reply:
Ya ... my point is these are **not blocks of code** and having people blindly copy and paste more than one line into a terminal is a bad practice.

I understand that's my preference though, I backed out that commit from this PR.